pages/blog/caching-data-with-dataloader.mdx

```diff
@@ -0,0 +1,137 @@
+import withArticle from '../../ui/blog/article'
+export const meta = {
+  title: 'Caching data with DataLoader',
+  author: 'giladtidhar',
+  tags: ['cache', 'data', 'servers', 'graphql'],
+  date: '2022-1-12',
+  description: 'How to use DataLoader to cache data',
+  image: '/blog-assets/caching-data-with-dataloader/cover.png'
+}
+export default withArticle({ ...meta })
+
+# Caching
+
+Caching data is an important thing, it could be the difference between a slow wasteful server and a fast efficient server.
+
+But, what is caching?
+A cache is a place in memory for important data, which can be delivered to the client faster than the regular procedure.  
+The goal of caching is to improve performance where it’s needed the most.
+
+## how is caching faster?
+
+Using a cache for important data is faster because of a few reasons:
+
+- A cache is usually not stored in storage, but in memory, which is a few times faster.
+- A cache is smaller than a database, so there are a lot less values to filter through.
+- Data that is queried a lot of times will be fetched from the database only one time, so there's less time lost for fetching multiple times.
+
+# how do we use caching?
+
+## About dataloader
+
+My prefered platform for caching is dataloader, and in this tutorial I will explain how to use dataloader with Node.js and Typescript, to have caching in your server.
+
+### A small caveat:
+
+The dataloader package was created specifically for GraphQL, but it can work with other Node.js programs.
+
```

Comment:
```suggestion
:::tip `dataloader` outside of GraphQL 

The `dataloader` package was created specifically for GraphQL, but it can work with other Node.js programs.

:::
```